app/assets/javascripts/qz/ticket-popup.js

```diff
@@ -0,0 +1,85 @@
+var privateKey = "-----BEGIN RSA PRIVATE KEY-----\n" +
```

Comment:
I'm going try and get this through anyway, but I have created a new issue to hold the conversation about this particular point: #1484